web/src/components/layout/scaffold/Tab.vue

```diff
@@ -11,7 +11,8 @@ import type { IconNames } from '~/components/atomic/Icon.vue';
 import { useTabsClient, type Tab } from '~/compositions/useTabs';
 
 const props = defineProps<{
-  id?: string;
+  to?: string;
+  alternativeRoute?: string;
```

Comment:
`$route.matched.some(({ name }) => name === 'customer_users')"` is probably what you were looking for?

Reply:
Just copied that sample over. You could check if a route is matching (should work for your question above as well) using `const route = useRoute()` With `route.matched`.